substrate/client/service/src/builder.rs

```diff
@@ -780,33 +783,35 @@ where
 	Ok(rpc_api)
 }
 
-/// Parameters to pass into `build_network`.
-pub struct BuildNetworkParams<'a, Block, Net, TxPool, IQ, Client>
-where
-	Block: BlockT,
-	Net: NetworkBackend<Block, <Block as BlockT>::Hash>,
-{
+/// Parameters to pass into [`build_network`].
+pub struct BuildNetworkParams<
+	'a,
+	TBl: BlockT,
```

Comment:
nit: I would keep naming consistent with `BuildNetworkAdvancedParams`, for example `Block` instead of `TBl`